.github/workflows/merge-pr.yml

```diff
@@ -64,6 +90,8 @@ jobs:
           NODE_AUTH_TOKEN: ${{secrets.NODE_AUTH_TOKEN}}
 
   calc-code-coverage:
+    needs: changes
+    if: needs.changes.outputs.rust == 'true'
```

Comment:
This needs to run on every merge or CodeCov has a hard time producing reports on changes.